cli/src/config/templates.toml

```diff
@@ -237,6 +246,17 @@ if(ref.tracking_present(), surround("(", ")", separate(", ",
 'format_snapshot_operation(op)' = 'format_operation(op)'
 'format_root_operation(root)' = 'separate(" ", root.id().short(), label("root", "root()")) ++ "\n"'
 
+
+'format_operation_oneline(op)' = '''
+  concat(
+    separate(" ", op.id().short(), op.user(), format_time_range(op.time())), " ",
+    op.description().first_line(), " ",
+    if(op.tags(), op.tags() ++ "\n"),
+  )
+'''
+'format_snapshot_operation_oneline(op)' = 'format_operation_oneline(op)'
+'format_root_operation_oneline(root)' = 'separate(" ", root.id().short(), label("root", "root()")) ++ "\n"'
```

Comment:
nit: might be okay to forward to `format_root_operation()` or not add "oneline" version at all. It's unlikely that the root operation template is customized to emit multi-line texts.

Reply:
done